### PR TITLE
[TASK] Improve Condition page

### PR DESCRIPTION
**Pull request by linawolf**
- use confvals
- correct indentation
- unify and improve examples (always end a condition, we are in TSconfig here)
- give titles to examples
- add todos where examples seemed strange to me

releases: main, 12.4, 11.5

### CODE REVIEW

File: Documentation/UsingSetting/Conditions.rst [hunk 1]
```diff
@@ -24,757 +24,868 @@ The Symfony expression language tends to throw warnings when sub arrays are
...
 tree.rootLine
 -------------
 
-:aspect:`Variable`
-   tree.rootLine
+..  confval:: tree.rootLine
+    :name: condition-tree-rootLine
+    :type: array
+
+    array of arrays with uid and pid. Only available in page TSconfig, not
```
Comment (brotkrueml) [inline Documentation/UsingSetting/Conditions.rst:212]:
```suggestion
    An array of arrays with uid and pid. Only available in page TSconfig, not
```

File: Documentation/UsingSetting/Conditions.rst [hunk 2]
```diff
@@ -24,757 +24,868 @@ The Symfony expression language tends to throw warnings when sub arrays are
...
 
-:aspect:`Variable`
-   tree.rootLineIds
+..  confval:: tree.rootLineIds
+    :name: condition-tree-rootLineIds
+    :type: array
 
-:aspect:`Type`
-   Array
+    An array with UIDs of the rootline. Only available in page TSconfig, not
```
Comment (brotkrueml) [inline Documentation/UsingSetting/Conditions.rst:240]:
```suggestion
    An array with UIDs of the root line. Only available in page TSconfig, not
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]
We use capitalized "[END]" in TypoScript conditions:
https://docs.typo3.org/m/typo3/reference-typoscript/12.4/en-us/Conditions/Index.html

Is this does not really matter we should use also capitalized "[END]" here for constistency reasons.

Additionally, this PR does many things, needed approx. 30 minutes for a review. Consider breaking them up into separate PRs in the future. I assume, will need the next time for this also 30 minutes :-(

File: Documentation/UsingSetting/Conditions.rst [hunk 3]
```diff
@@ -24,757 +24,868 @@ The Symfony expression language tends to throw warnings when sub arrays are
...
+
+    UID of current workspace.
 
-:aspect:`Type`
-   Integer
+..  _condition-workspace-workspaceId-example:
 
-:aspect:`Description`
-   UID of current workspace.
+Example: Condition applies only in a certain workspace
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/UsingSetting/Conditions.rst:469]: Here we can use any workspace ID while the example bellow checks if we are in the life workspace. So I would keep the text as it currently is

**Comment by linawolf** [comment]
I changed [end] into [END] to be consistent, however [end] also works

**Comment by brotkrueml** [comment]
> I changed [end] into [END] to be consistent, however [end] also works

Yup, both work, also [global] and [GLOBAL], but we should be consistent throughout the docs.

File: Documentation/UsingSetting/Conditions.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/UsingSetting/Conditions.rst:469]: Then maybe change the example to an arbitrary number (42).

**Comment by linawolf** [comment]
So are there any other open issues?

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**